site/source/docs/api_reference/val.h.rst

```diff
@@ -262,6 +262,43 @@ Guide material for this class can be found in :ref:`embind-val-guide`.
 
       .. note:: This method requires :ref:`Asyncify` to be enabled.
 
+  .. cpp:function:: val operator co_await() const
+
+   The ``co_await`` operator allows to await JavaScript promises represented by ``val``.
+
+   It's compatible with any C++20 coroutines, but should be normally used inside
+   a ``val``-returning coroutine which will also become a ``Promise``.
+
+   For example, it allows to implement an equivalent of this JavaScript ``async``/``await`` function:
```

Comment:
Change to: For example, it allows you to implement the equivalent of this JavaScript...

Reply:
```suggestion
   For example, it allows you to implement the equivalent of this JavaScript ``async``/``await`` function:
```